gapic/api/artman_clouddebugger.yaml

```diff
@@ -0,0 +1,30 @@
+common:
+  api_name: google-clouddebugger-v2
+  proto_gen_pkg_deps:
+    - google-common-protos
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2
+  service_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/clouddebugger.yaml
+  gapic_api_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml
+  output_dir: ${REPOROOT}/artman/output
+  enable_batch_generation: True
+java:
+  final_repo_dir: ${REPOROOT}/google-cloud-java/google-cloud-clouddebugger
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-clouddebugger
+go:
+  final_repo_dir: ${REPOROOT}/gapi-clouddebugger-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-clouddebugger
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-clouddebugger
+ruby:
+  final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-clouddebugger
+  skip_packman: True
```

Comment:
> There will always be some configuration required for each API we generate.

Reactive corrections to problems identified in the generated library are fine. Configuration to be set (and reviewed) before the library can be generated is problematic, for being hard to scale.

> The cost of creating new handwritten client libraries is very high compared to the cost of manually setting this flag.

I'm not worried about adding a small cost to creating a new handwritten library. I'm worried about an engineer having to check whether a handwritten library exists or not before being able to autogenerate a library. We want to be able to generate libraries for hundreds of APIs (and, ideally, an unbound amount): every cost in the process needs to have that multiplier applied before considering its magnitude.

> There is a small, finite number of existing APIs that have handwritten client libraries, for which this flag needs to be set

So a machine-readable list should be easy to create? :)

But even so, I think the decision should be made at a later stage: Say Jun or a computer verifies that there's no hand-written library for this API today, and we proceed all the way with the generation. What happens when some time in the future they want to write a hand-written library on top?